src/apps/api/views/submissions.py

```diff
@@ -175,10 +209,12 @@ def upload_submission_scores(request, submission_pk):
 @api_view(('GET',))
 def can_make_submission(request, phase_id):
     phase = get_object_or_404(Phase, id=phase_id)
-    user_is_approved = phase.competition.participants.filter(user=request.user, status=CompetitionParticipant.APPROVED).exists()
+    user_is_approved = phase.competition.participants.filter(user=request.user,
+                                                             status=CompetitionParticipant.APPROVED).exists()
```

Comment:
```suggestion
    user_is_approved = phase.competition.participants.filter(
        user=request.user,
        status=CompetitionParticipant.APPROVED,
    ).exists()
```

If we change this to my suggestion, we can use it as a repeatable solution to lines that are too long.